app/views/weight_estimator/_household_goods.html.erb

```diff
@@ -10,11 +10,11 @@
       <%- household_goods.each do |household_good| -%>
         <tr>
           <td>
-            <label for="<%= household_good.key %>"><%= household_good.name %></label>
+            <%= label_tag household_good.key, household_good.name %>
```

Comment:
Here and elsewhere in these views, I flipped over to Rails view helpers to clean things up a bit.